src/main/java/com/mongodb/jdbc/MongoJsonSchema.java

```diff
@@ -0,0 +1,67 @@
+package com.mongodb.jdbc;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public class MongoJsonSchema {
+    public String bsonType;
+    public Map<String, MongoJsonSchema> properties;
+    public Set<MongoJsonSchema> anyOf;
+    public Set<String> required;
+    public MongoJsonSchema items;
+    public boolean additionalProperties;
+
+    public MongoJsonSchema() {}
+
+    public static MongoJsonSchema createEmptyObjectSchema() {
+        MongoJsonSchema ret = new MongoJsonSchema();
+        ret.bsonType = "object";
+        ret.properties = new HashMap<String, MongoJsonSchema>();
+        ret.required = new HashSet<String>();
+        return ret;
+    }
+
+    public static MongoJsonSchema createScalarSchema(String type) {
+        MongoJsonSchema ret = new MongoJsonSchema();
+        ret.bsonType = type;
+        return ret;
+    }
+
+    @SafeVarargs
+    public final void addRequiredScalarKeys(Pair<String, String>... values) {
```

Comment:
Can you add a comment to this method explaining what the arguments are? It is clear enough from reading the code, but I wonder if that's because I have more familiarity with the codebase than a newcomer might.

I don't mind if the comment isn't javadoc style, but I'm not sure what java convention is in that department. Maybe a javadoc style comment would be best.